packages/zowe-explorer/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 - Fixed the allocate-like functionality by removing the inclusion of DS item in the filter history. [#2620](https://github.com/zowe/vscode-extension-for-zowe/issues/2620)
 - Fixed issue with `Submit JCL` losing focus on JCL being submitted, causing the wrong job submission. [#2616](https://github.com/zowe/vscode-extension-for-zowe/issues/2616)
+- Update dependencies to address vulnerabilities.
```

Comment:
Do we want to call out the specific version where vulnerabilities were fixed, or should we just say something along the lines of "Updating dependencies for technical currency"?

Reply:
working on PRs for other branches with the versions Timothy shared